src/main/resources/META-INF/plugin.xml

```diff
@@ -23,6 +23,13 @@
     <editorFactoryListener implementation="snyk.advisor.AdvisorEditorFactoryListener"/>
 
     <errorHandler implementation="snyk.errorHandler.SnykErrorReportSubmitter"/>
+
+    <registryKey
+      key="snyk.timeout.results.waiting"
+      defaultValue="720000"
+      description="Snyk timeout (milliseconds) to wait for results during scan"
```

Comment:
@ArtsiomCh, let set `restartRequired=true` parameter too.

Reply:
why do we need restart here? we request that value dynamically through getWaitForResultsTimeout()
So no restart needed. Am I missing something?

Reply:
right, our settings-bug put me into "always-restart" mode :).